examples/llms/providers/watsonx.ts

```diff
@@ -2,7 +2,7 @@ import "dotenv/config";
 import { BaseMessage } from "bee-agent-framework/llms/primitives/message";
 import { WatsonXChatLLM } from "bee-agent-framework/adapters/watsonx/chat";
 
-const chatLLM = WatsonXChatLLM.fromPreset("meta-llama/llama-3-1-70b-instruct", {
+const chatLLM = WatsonXChatLLM.fromPreset("meta-llama/Llama-3.2-3B-Instruct", {
```

Comment:
The name is incorrect. The correct one should be `llama-3-2-3b-instruct`.
Please do a test run to ensure whether it works.